docs/UserGuide.adoc

```diff
@@ -10,10 +10,11 @@
 ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
+:important-caption: :white_check_mark:
 endif::[]
 :repoURL: https://github.com/CS2103JAN2018-F14-B1/main
 
-By: `F14-B1`      Since: `Mar 2018`      Licence: `MIT`
+By: `F14-B1`      Since: `Jun 2016`      Licence: `MIT`
```

Comment:
should this be mar 2018 instead? since it is when we start working on our project